ds_fastapi_middleware/utils/internal_traffic.py

```diff
@@ -3,17 +3,14 @@
 import struct
 import socket
 
-from ds_fastapi_middleware.config import Config
 
-
```

Comment:
Should we maybe add a default list of networks here? I'm slightly concerned about defining the network list on every service using the middleware, presumably we are going to use the same list more or less across all of them?

I'm thinking we could do something like 
`DEFAULT_NETWORKS=[....]

...

from utils.internal_traffic import DEFAULT_NETWORKS
app.add_middleware(AuditMiddleware, logger=audit_logger, networks=DEFAULT_NETWORKS)
`

I guess with this approach we could also build a modified list if required based on the default.